dash-pipeline/Makefile

```diff
@@ -278,18 +290,18 @@ saithrift-server-clean: docker-saithrift-bldr-image-exists
 # SAI TEST TARGETS
 ###############################
 
-.PHONY:test libsai-test
-test:libsai-test
+.PHONY:test
+test: libsai-test
 
+.PHONY:test-clean
 test-clean:
 	make -C tests/libsai clean
 
-
 .PHONY:libsai-test
-libsai-test:SAI/SAI SAI/lib/libsai.so
-	chmod -R a+w tests/libsai
+libsai-test: SAI/lib/libsai.so
```

Comment:
The `SAI/SAI` dependency ensures SAI submodule gets reinstalled in case it's missing or you deliberately delete it to clean up some problems. I'm not sure if you remove this dependency that it'll reinstall it.

Reply:
i can bring this back

Reply:
ok, i i can't bring it back, since if SAI/SAI will be executed, currently generated headers will be overrided by sai_gen_api.py each time, and we want to avoid that to not recompile libsai.so each time, even if there were no changes, if you want to regenerate headers  call "make libsai-clean" or "make clean", that clean has depenedency on SAI/SAI https://github.com/sonic-net/DASH/pull/432/files#diff-7bbbcc308b956ce1e66a2922daf602dfe4680ac7692d0a0e51270fd4939d69dbL205 and will properly regenerate headers